src/main/java/use_cases/BaseStockInteractor.java

```diff
@@ -33,10 +31,10 @@ protected void updatePortfolio(User user, String ticker, Double amount) {
     }
 
     protected void addToHistory(HashMap<String, TransactionHistory> userHistory, String ticker,
-                                User user, Double amount, Double currentPrice, Transaction transaction) {
```

Comment:
why remove amount from the function?

Reply:
was never used, and the passed-in transaction filled that role as it had a price